sim/druid/runes.go

```diff
@@ -25,7 +25,6 @@ func (druid *Druid) ApplyRunes() {
 	druid.applyElunesFires()
 
 	// Hands
-	druid.applyMangle()
```

Comment:
Removing this will make the sim not register the mangle cat spell I think. I would leave this here but remove the parts in `druid.go` that call `registerMangleBearSpell` since mangle is a rune in SOD